Classes/Repository/RepositoryEmptyResultsSectionController.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  RepositoryEmptyResultsSectionController.swift
+//  Freetime
+//
+//  Created by Sherlock, James on 01/08/2017.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import IGListKit
+
+final class RepositoryEmptyResultsSectionController: ListSectionController {
+    
+    let topInset: CGFloat
+    let topLayoutGuide: UILayoutSupport
+    let type: EmptyResultsType
+    
+    init(topInset: CGFloat, topLayoutGuide: UILayoutSupport, type: EmptyResultsType) {
+        self.topInset = topInset
+        self.topLayoutGuide = topLayoutGuide
+        self.type = type
+        super.init()
+    }
+    
+    override func sizeForItem(at index: Int) -> CGSize {
+        guard let size = collectionContext?.containerSize else { fatalError("Missing context") }
+        return CGSize(width: size.width, height: size.height - topInset - topLayoutGuide.length)
+    }
+    
+    override func cellForItem(at index: Int) -> UICollectionViewCell {
+        guard let cell = collectionContext?.dequeueReusableCell(of: RepositoryEmptyResultsCell.self, for: self, at: index) as? RepositoryEmptyResultsCell else {
+                fatalError("Missing context, object, or cell is wrong type")
```

Comment:
Correct, originally else was on it's own line which would've meant the indentation would be acceptable but in it's currently form yea